src/main/scala/beam/router/BeamRouter.scala

```diff
@@ -126,10 +131,29 @@ class BeamRouter(
 
   private var traveTimeOpt: Option[TravelTime] = None
 
+  val kryoSerializer = new KryoSerializer(context.system.asInstanceOf[ExtendedActorSystem])
+
+  private val updateTravelTimeTimeout: Timeout = Timeout(3, TimeUnit.MINUTES)
+
   override def receive: PartialFunction[Any, Unit] = {
     case `tick` =>
       if (isWorkAndNoAvailableWorkers) notifyWorkersOfAvailableWork()
       logExcessiveOutstandingWorkAndClearIfEnabledAndOver
+    case t: TryToSerialize =>
```

Comment:
This part seems like unnecessary overhead just for logging purposes. Maybe we could put it behind a flag which defaults to off?

Reply:
I left this here to see what is the size of payload. In case if it's not able to serialize it due to the limitation of message size. I'll add some flag to do it only on debug level